v0/destinations/hs/util.js

```diff
@@ -375,7 +375,7 @@ const getEventAndPropertiesFromConfig = (message, destination, payload) => {
   }
 
   // 2. fetch event properties from webapp config
-  eventProperties = getHashFromArray(eventProperties);
+  eventProperties = getHashFromArray(eventProperties, ...Array(2), false);
```

Comment:
Will the contents of `...Array(2)` will be undefined?

Reply:
Yes